scripts/change-fonts-cjk.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/sh
+set -e
+
+FONTDIR="./fonts"
+prevcjk=""
+
+mkdir -p "${FONTDIR}"
+
+# download filename url
+download() {
+  ## Download if newer, and if curl fails, clean up and exit
+  curl --fail --compressed -A "get-fonts.sh/osm-carto" -o "$1" -z "$1" -L "$2" || { echo "Failed to download $1 $2"; rm -f "$1"; exit 1; }
+}
+
+CJKLOCALES="jp
+kr
+tc
+sc
+hk"
+
+echo "--------------------------------------"
+echo -n "1. Japanese (default)"
+if [ -e "./fonts/NotoSansCJKjp-Regular.otf" ]; then
+  echo " [downloaded]"
+  prevcjk=jp
+else
+  echo ""
+fi
+
+echo -n "2. Korean"
+if [ -e "./fonts/NotoSansCJKkr-Regular.otf" ]; then
+  echo " [downloaded]"
+  prevcjk=kr
+else
+  echo ""
+fi
+
+echo -n "3. Simplified Chinese"
+if [ -e "./fonts/NotoSansCJKsc-Regular.otf" ]; then
+  echo " [downloaded]"
+  prevcjk=sc
+else
+  echo ""
+fi
+
+echo -n "4. Traditional Chinese"
+if [ -e "./fonts/NotoSansCJKtc-Regular.otf" ]; then
+  echo " [downloaded]"
+  prevcjk=tc
+else
+  echo ""
+fi
+
+echo -n "5. Traditional Chinese (Hong Kong)"
+if [ -e "./fonts/NotoSansCJKhk-Regular.otf" ]; then
+  echo " [downloaded]"
+  prevcjk=hk
+else
+  echo ""
+fi
+echo "--------------------------------------"
+
+while true; do
+  read -p "Enter which CJK language is prefered above [1-5]: " cjklang
```

Comment:
I'd rather see the font selection be a command-line option than reading from stdin. This makes it easier to script.

Reply:
How about `scripts/change-fonts-cjk.sh japanese`?